book/chrome.md

```diff
@@ -93,8 +93,8 @@ their own `layout` and `paint` methods, but before we get to those we
 need to think about how the `LineLayout` and `TextLayout` objects will
 be created. That has to happen during word wrapping.
 
-Recall [how word wrapping happens (see Chapter 3)](text.md)
-inside `BlockLayout`'s `word` method. That method updates a `line` field,
+Recall [how word wrapping (see Chapter 3)](text.md)
+inside `BlockLayout`'s `word` method works. That method updates a `line` field,
```

Comment:
Otherwise I parsed "Recall how X happens" as meaning the same thing as "Recall that X happens" and that made the next sentence weird.